benchmark.js

```diff
@@ -131,7 +131,10 @@ var Benchmark = (function() {
   }
 
   var valueFormatters = {
-    startupTime: msFormatter,
+    totalStartupTime: msFormatter,
+    vmStartupTime: msFormatter,
+    backgroundStartupTime: msFormatter,
+    foregroundStartupTime: msFormatter,
```

Comment:
Nit: I would actually call these *bgStartupTime* and *fgStartupTime*, as in your example, to conserve the width of the table.